css/index.css

```diff
@@ -4181,10 +4182,14 @@ a.text-dark:focus, a.text-dark:hover {
   .meetup__summary__title {
     font-size: 35px; }
   .meetup__summary__text {
-    font-weight: 800;
+    font-weight: 500;
     font-size: 20px; }
   .meetup__summary__photo {
     background-color: white; }
+  .meetup__summary .meetup-img-fluid {
+    height: 400;
+    width: 540;
+    margin: 60px; }
```

Comment:
Is it really necessary? We already set these styles on `_meetups.scss`. :thinking:

Reply:
Oh hell, I think all CSS files should be in `.gitignore` file - we're generating it from SASS.